### ISSUE TITLE
Pod web keeps restarting when started as non-root

### ISSUE THREAD

**Issue by michaelowens** [yyx990803/pod#41]
It shouldn't need root for the default port, but for some reason the restarts just skyrocketing. The problem is that I create my pod apps as my own user, so running the web interface as root will work but not show any pod-apps...

![Restarts](http://i.imgur.com/VAaazV3.png)

**Comment by yyx990803** [comment]
hmm, you should start pod web using the same user that runs the apps, because they need to share the same `.podrc` in the home folder.

**Comment by michaelowens** [comment]
Yeah, that's why I don't get my list of apps when running it as root, because I create apps *without* `sudo`, but the web client won't run without it.

**Comment by yyx990803** [comment]
what happens when you run `pod web` without sudo?

**Comment by michaelowens** [comment]
![pod web](http://i.imgur.com/7MUwLlv.png)

**Comment by michaelowens** [comment]
Hmm, changed the port and now it does work.
